src/otio_aaf_adapter/adapters/advanced_authoring_format.py

```diff
@@ -868,8 +868,8 @@ def _transcribe(item, parents, edit_rate, indent=0):
             elif media_kind in ("SoundMasterTrack", "Sound"):
                 result.kind = otio.schema.TrackKind.Audio
             else:
-                # Timecode, Edgecode, others?
-                result.kind = ""
+                # Timecode, Edgecode, Data, ...
+                result.kind = f"AAF_{media_kind}"
```

Comment:
For reference, [here are the DataDefs](https://github.com/markreidvfx/pyaaf2/blob/main/src/aaf2/model/datadefs.py#L101C7-L101C7) that drive the media kind that's used from @markreidvfx